Dockerfile

```diff
@@ -29,10 +29,14 @@ FROM ubuntu:22.04 as runtime
 
 WORKDIR /
 
-RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y ca-certificates
+# curl is needed for healthcheck
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y ca-certificates curl
 
 COPY --from=build /app/target/release/bors .
 
 EXPOSE 80
 
+HEALTHCHECK --timeout=10s --start-period=1s \
```

Comment:
I'm not familiar with Docker healthchecks, what does this actually do? Or rather, I understand what it does, but why do we need it?

Reply:
To fix https://github.com/rust-lang/bors/issues/121

AWS uses this to check if the app is ready and it's running fine. E.g. it might be stuck due to high traffic or not ready yet.

Reply:
I see, so AWS uses the `HEALTHCHECK` machinery from the Dockerfile? I thought that it just hardcores the `/health` endpoint :)

Reply:
yes. this is my understanding from the docs: https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_HealthCheck.html

Reply:
Ok. I would increase the start period from 1s to 10s, otherwise LGTM.